src/Type/WPInterfaceType.php

```diff
@@ -98,6 +98,15 @@ public function prepare_fields( $fields, $type_name ) {
 		 * as it ensures it's not output in just random order
 		 */
 		ksort( $fields );
+
```

Comment:
Can you explain why this filter is being applied here? It appears to be merging fields with themselves. This seems unnecessary, but perhaps I'm missing something?

Reply:
To catch all fields registered with `graphql_register_field()` on the Interface.